packages/mui-utils/src/deepmerge.ts

```diff
@@ -1,3 +1,5 @@
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
lodash is a no go.

```suggestion
```

Reply:
@oliviertassinari I'm agree, it have been added to illustrate an solution and check that all tests will be passed. Should we write own implementation of clonedeep or do we have other ways?